src/states_screens/feature_unlocked.cpp

```diff
@@ -224,26 +225,41 @@ void FeatureUnlockedCutScene::findWhatWasUnlocked(RaceManager::Difficulty diffic
     {
         addUnlockedGP(grand_prix_manager->getGrandPrix(gps[i]));
     }
+    for (unsigned int i = 0; i < karts.size(); i++)
+    {
+        addUnlockedKart(const_cast<KartProperties*>(kart_properties_manager->getKart(karts[i])));
```

Comment:
Can const_cast be avoided? That's one part of c++ that's usually best left alone

A better fix would probably be to modify addUnlockedKart to receive a const KartProperties* parameter